src/views/Applications/ApplicationDashboard.vue

```diff
@@ -319,15 +321,25 @@ onMounted(() => {
 </script>
 
 <style lang="scss" scoped>
-@import '../../assets/variables.scss';
+
+.application-dashboard-wrapper {
+  .page-title {
+    margin-bottom: 24px;
```

Comment:
We want to use the design token values e.g. 

```
$kui-space-80: 24px; // 24px
```

There are design tokens for other sizes / font sizes as well too.

Reply:
Oh got it, I didn't see reference to the tokens yet, so I figured that was a separate phase!